rslib/ascii_percent_encoding/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "ascii_percent_encoding"
+publish = false
+description = "Like percent_encoding, but does not encode non-ASCII characters."
+
+version.workspace = true
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
```

Comment:
We should probably define authors and license using the original crate's values rather than using our own names/license here.